docs/reference/api-overview.md

```diff
@@ -8,55 +8,64 @@ approvers:
+{: .note}
 
-The criteria are summarized here:
+Different API versions imply different levels of stability and support. You can find more information about the criteria for each level in the [API Changes documentation](https://git.k8s.io/community/contributors/devel/api_changes.md#alpha-beta-and-stable-versions).  
 
-- Alpha level:
+Here's a summary of each level:
+- Alpha:
   - The version names contain `alpha` (for example, `v1alpha1`).
-  - The software may contain bugs. Enabling a feature may expose bugs.  A feature may be disabled by default.
+  - The software may contain bugs. Enabling a feature may expose bugs. A feature may be disabled by default.
   - The support for a feature may be dropped at any time without notice.
   - The API may change in incompatible ways in a later software release without notice.
-  - The software is recommended for use only in short-lived testing clusters, due to increased risk of bugs and lack of long-term support.
-- Beta level:
+  - The software is recommended for use only in short-lived testing clusters, due to increased risk of bugs and lack of   long-term support.
+  
+- Beta:
   - The version names contain `beta` (for example, `v2beta3`).
-  - The software is well tested.  Enabling a feature is considered safe.  Features are enabled by default.
+  - The software is well tested. Enabling a feature is considered safe. Features are enabled by default.
   - The support for a feature will not be dropped, though the details may change.
-  - The schema and/or semantics of objects may change in incompatible ways in a subsequent beta or stable release.  When this happens, migration instructions will be provided.  This may require deleting, editing, and re-creating
+  - The schema and/or semantics of objects may change in incompatible ways in a subsequent beta or stable release. When this happens, migration instructions are provided.  This may require deleting, editing, and re-creating
     API objects. The editing process may require some thought. This may require downtime for applications that rely on the feature.
-  - The software is recommended for only non-business-critical uses because of potential for incompatible changes in subsequent releases.  If you have multiple clusters which can be upgraded independently, you may be able to relax this restriction.
-  - **Please try our beta features and give feedback on them!  Once they exit beta, it may not be practical for us to make more changes.**
-- Stable level:
+  - The software is recommended for only non-business-critical uses because of potential for incompatible changes in subsequent releases. If you have multiple clusters which can be upgraded independently, you may be able to relax this restriction.</br>
+  
+  **Note:** Try the beta features and provide feedback. After the features exit beta, it may not be practical to make more changes.
```

Comment:
This is rendering somewhat oddly at https://deploy-preview-7817--kubernetes-io-master-staging.netlify.com/docs/reference/api-overview/ - and I think it's this **Note:** segment that's indented a single space. Jekyll seems to be linking it to the above lists and including that in a giant call-out